src/liballoc/vec.rs

```diff
@@ -736,13 +736,13 @@ impl<T> Vec<T> {
     pub fn truncate(&mut self, len: usize) {
         // This is safe because:
         //
-        // * the slice passed to `drop_in_place` is valid; the `len > self.len`
+        // * the slice passed to `drop_in_place` is valid; the `len >= self.len`
         //   case avoids creating an invalid slice, and
         // * the `len` of the vector is shrunk before calling `drop_in_place`,
         //   such that no value will be dropped twice in case `drop_in_place`
         //   were to panic once (if it panics twice, the program aborts).
         unsafe {
-            if len > self.len {
+            if len >= self.len {
```

Comment:
@ecstatic-morse Can you please check on this? I think this also impacts `resize` where it runs truncate even though the length is the same. CC @lzutao 

```rust
pub fn resize(&mut self, new_len: usize, value: T) {
    let len = self.len();

    if new_len > len {
        self.extend_with(new_len - len, ExtendElement(value))
    } else {
        self.truncate(new_len); // truncate run the rest of the code even though the length is same
    }
}
```

Reply:
@pickfire What would you like me to do exactly? This PR was closed because the reviewer refused to sign off on it. Talk to them.

Reply:
@ecstatic-morse The reviewer is leaving the team already. See the message on zulip.

Reply:
That's too bad. Still, I don't see what you want from me. While I think this PR is fine as-is and was held to an unusually high standard during the review process, stepping away from the project does not invalidate one's opinion. If you can find someone with r+ rights who feels strongly that these changes are correct, then I guess we can reopen this, but I'm not going  to try to cherry pick a reviewer.